filestore/handlers.py

```diff
@@ -446,3 +446,31 @@ def __init__(self, filename):
 
     def __call__(self):
         return tifffile.imread(self._name)
+
+
+class DATHandler(HandlerBase):
+    ''' This handles vague text files through numpy's loadtxt module.
+            Useful for the ubiquitous, vague and well loved ".DAT" file.
+            Uses defaults from np.loadtxt.
+
+        See Also
+        --------
+        np.loadtxt
+    '''
+    specs = {'DAT'} | HandlerBase.specs
+    allowed_kwargs = ['comments', 'delimiter', 'converters', 'skiprows',
+                      'usecols', 'unpack', 'ndmin']
+
+    def __init__(self, fpath, **kwargs):
+        self._path = fpath
+        # parse kwargs to make sure np.loadtxt can handle them, ignore rest
+        # TODO : maybe warn if there are extra arguments? (symptom of improper
+        # data handling?)
+        newkwargs = dict()
```

Comment:
I would drop this validation all together.  The goal at this level is to be as _unforgiving_ as possible.